tests/phpunit/tests/rest-api/rest-autosaves-controller.php

```diff
@@ -674,4 +674,38 @@ public function test_rest_autosave_draft_post_locked_to_different_author() {
 
 		wp_delete_post( $post_id );
 	}
+
+	/**
+	 * @ticket 49532
+	 */
+	public function test_rest_autosave_do_not_create_duplicate_revision_when_it_call_multiple_times() {
```

Comment:
```suggestion
	public function test_rest_autosave_do_not_create_autosave_when_content_unchanged() {
```
I think this more closely captures the intent behind the fix.

Reply:
Function name updated.